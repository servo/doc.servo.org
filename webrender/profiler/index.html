<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><meta name="generator" content="rustdoc"><meta name="description" content="API documentation for the Rust `profiler` mod in crate `webrender`."><meta name="keywords" content="rust, rustlang, rust-lang, profiler"><title>webrender::profiler - Rust</title><link rel="stylesheet" type="text/css" href="../../normalize.css"><link rel="stylesheet" type="text/css" href="../../rustdoc.css" id="mainThemeStyle"><link rel="stylesheet" type="text/css" href="../../light.css"  id="themeStyle"><link rel="stylesheet" type="text/css" href="../../dark.css" disabled ><link rel="stylesheet" type="text/css" href="../../ayu.css" disabled ><script id="default-settings" ></script><script src="../../storage.js"></script><script src="../../crates.js"></script><noscript><link rel="stylesheet" href="../../noscript.css"></noscript><link rel="icon" type="image/svg+xml" href="../../favicon.svg"><link rel="alternate icon" type="image/png" href="../../favicon-16x16.png"><link rel="alternate icon" type="image/png" href="../../favicon-32x32.png"><style type="text/css">#crate-search{background-image:url("../../down-arrow.svg");}</style></head><body class="rustdoc mod"><!--[if lte IE 11]><div class="warning">This old browser is unsupported and will most likely display funky things.</div><![endif]--><nav class="sidebar"><div class="sidebar-menu" role="button">&#9776;</div><a href='../../webrender/index.html'><div class='logo-container rust-logo'><img src='../../rust-logo.png' alt='logo'></div></a><h2 class="location">Module profiler</h2><div class="sidebar-elems"><div class="block items"><ul><li><a href="#modules">Modules</a></li><li><a href="#structs">Structs</a></li><li><a href="#enums">Enums</a></li><li><a href="#constants">Constants</a></li><li><a href="#statics">Statics</a></li><li><a href="#traits">Traits</a></li><li><a href="#functions">Functions</a></li></ul></div><div id="sidebar-vars" data-name="profiler" data-ty="mod" data-relpath="./"></div><script defer src="./sidebar-items.js"></script></div></nav><div class="theme-picker"><button id="theme-picker" aria-label="Pick another theme!" aria-haspopup="menu" title="themes"><img width="18" height="18" alt="Pick another theme!" src="../../brush.svg"></button><div id="theme-choices" role="menu"></div></div><nav class="sub"><form class="search-form"><div class="search-container"><div><select id="crate-search"><option value="All crates">All crates</option></select><input class="search-input" name="search" disabled autocomplete="off" spellcheck="false" placeholder="Click or press ‘S’ to search, ‘?’ for more options…" type="search"></div><button type="button" id="help-button" title="help">?</button><a id="settings-menu" href="../../settings.html" title="settings"><img width="18" height="18" alt="Change settings" src="../../wheel.svg"></a></div></form></nav><section id="main" class="content"><h1 class="fqn"><span class="in-band">Module <a href="../index.html">webrender</a>::<wbr><a class="mod" href="#">profiler</a><button id="copy-path" onclick="copy_path(this)" title="copy path"><img src="../../clipboard.svg" width="19" height="18" alt="Copy item import" title="Copy item import to clipboard"></button></span><span class="out-of-band"><span id="render-detail"><a id="toggle-all-docs" href="javascript:void(0)" title="collapse all docs">[<span class="inner">&#x2212;</span>]</a></span><a class="srclink" href="../../src/webrender/profiler.rs.html#5-1901" title="goto source code">[src]</a></span></h1><h2 id="modules" class="section-header"><a href="#modules">Modules</a></h2>
<div class="item-table"><div class="item-left module-item"><a class="mod" href="expected/index.html" title="webrender::profiler::expected mod">expected</a></div><div class="item-right docblock-short"></div></div><h2 id="structs" class="section-header"><a href="#structs">Structs</a></h2>
<div class="item-table"><div class="item-left module-item"><a class="struct" href="struct.AverageIntProfileCounter.html" title="webrender::profiler::AverageIntProfileCounter struct">AverageIntProfileCounter</a></div><div class="item-right docblock-short"><p>A profile counter recording average and maximum integer values over time slices
of half a second.</p>
</div><div class="item-left module-item"><a class="struct" href="struct.AverageTimeProfileCounter.html" title="webrender::profiler::AverageTimeProfileCounter struct">AverageTimeProfileCounter</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.BackendProfileCounters.html" title="webrender::profiler::BackendProfileCounters struct">BackendProfileCounters</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.ChangeIndicator.html" title="webrender::profiler::ChangeIndicator struct">ChangeIndicator</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.DrawState.html" title="webrender::profiler::DrawState struct">DrawState</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.FrameProfileCounters.html" title="webrender::profiler::FrameProfileCounters struct">FrameProfileCounters</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.GpuCacheProfileCounters.html" title="webrender::profiler::GpuCacheProfileCounters struct">GpuCacheProfileCounters</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.GpuFrame.html" title="webrender::profiler::GpuFrame struct">GpuFrame</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.GpuFrameCollection.html" title="webrender::profiler::GpuFrameCollection struct">GpuFrameCollection</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.GpuProfileTag.html" title="webrender::profiler::GpuProfileTag struct">GpuProfileTag</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.GraphStats.html" title="webrender::profiler::GraphStats struct">GraphStats</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.IntProfileCounter.html" title="webrender::profiler::IntProfileCounter struct">IntProfileCounter</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.InternProfileCounters.html" title="webrender::profiler::InternProfileCounters struct">InternProfileCounters</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.PercentageProfileCounter.html" title="webrender::profiler::PercentageProfileCounter struct">PercentageProfileCounter</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.ProfileGraph.html" title="webrender::profiler::ProfileGraph struct">ProfileGraph</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.ProfileScope.html" title="webrender::profiler::ProfileScope struct">ProfileScope</a></div><div class="item-right docblock-short"><p>A simple RAII style struct to manage a profile scope.</p>
</div><div class="item-left module-item"><a class="struct" href="struct.Profiler.html" title="webrender::profiler::Profiler struct">Profiler</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.RendererProfileCounters.html" title="webrender::profiler::RendererProfileCounters struct">RendererProfileCounters</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.RendererProfileTimers.html" title="webrender::profiler::RendererProfileTimers struct">RendererProfileTimers</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.ResourceProfileCounter.html" title="webrender::profiler::ResourceProfileCounter struct">ResourceProfileCounter</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.ResourceProfileCounters.html" title="webrender::profiler::ResourceProfileCounters struct">ResourceProfileCounters</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.TextureCacheProfileCounters.html" title="webrender::profiler::TextureCacheProfileCounters struct">TextureCacheProfileCounters</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.TimeProfileCounter.html" title="webrender::profiler::TimeProfileCounter struct">TimeProfileCounter</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.Timer.html" title="webrender::profiler::Timer struct">Timer</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="struct" href="struct.TransactionProfileCounters.html" title="webrender::profiler::TransactionProfileCounters struct">TransactionProfileCounters</a></div><div class="item-right docblock-short"></div></div><h2 id="enums" class="section-header"><a href="#enums">Enums</a></h2>
<div class="item-table"><div class="item-left module-item"><a class="enum" href="enum.ProfileStyle.html" title="webrender::profiler::ProfileStyle enum">ProfileStyle</a></div><div class="item-right docblock-short"></div></div><h2 id="constants" class="section-header"><a href="#constants">Constants</a></h2>
<div class="item-table"><div class="item-left module-item"><a class="constant" href="constant.AVERAGE_OVER_NS.html" title="webrender::profiler::AVERAGE_OVER_NS constant">AVERAGE_OVER_NS</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="constant" href="constant.GRAPH_FRAME_HEIGHT.html" title="webrender::profiler::GRAPH_FRAME_HEIGHT constant">GRAPH_FRAME_HEIGHT</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="constant" href="constant.GRAPH_HEIGHT.html" title="webrender::profiler::GRAPH_HEIGHT constant">GRAPH_HEIGHT</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="constant" href="constant.GRAPH_PADDING.html" title="webrender::profiler::GRAPH_PADDING constant">GRAPH_PADDING</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="constant" href="constant.GRAPH_WIDTH.html" title="webrender::profiler::GRAPH_WIDTH constant">GRAPH_WIDTH</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="constant" href="constant.ONE_SECOND_NS.html" title="webrender::profiler::ONE_SECOND_NS constant">ONE_SECOND_NS</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="constant" href="constant.PROFILE_PADDING.html" title="webrender::profiler::PROFILE_PADDING constant">PROFILE_PADDING</a></div><div class="item-right docblock-short"></div></div><h2 id="statics" class="section-header"><a href="#statics">Statics</a></h2>
<div class="item-table"><div class="item-left module-item"><a class="static" href="static.PROFILER_HOOKS.html" title="webrender::profiler::PROFILER_HOOKS static">PROFILER_HOOKS</a></div><div class="item-right docblock-short"><p>The current global profiler callbacks, if set by embedder.</p>
</div></div><h2 id="traits" class="section-header"><a href="#traits">Traits</a></h2>
<div class="item-table"><div class="item-left module-item"><a class="trait" href="trait.ProfileCounter.html" title="webrender::profiler::ProfileCounter trait">ProfileCounter</a></div><div class="item-right docblock-short"></div><div class="item-left module-item"><a class="trait" href="trait.ProfilerHooks.html" title="webrender::profiler::ProfilerHooks trait">ProfilerHooks</a></div><div class="item-right docblock-short"><p>Defines the interface for hooking up an external profiler to WR.</p>
</div></div><h2 id="functions" class="section-header"><a href="#functions">Functions</a></h2>
<div class="item-table"><div class="item-left module-item"><a class="fn" href="fn.add_event_marker.html" title="webrender::profiler::add_event_marker fn">add_event_marker</a></div><div class="item-right docblock-short"><p>Records a marker of the given duration that just ended.</p>
</div><div class="item-left module-item"><a class="fn" href="fn.add_text_marker.html" title="webrender::profiler::add_text_marker fn">add_text_marker</a></div><div class="item-right docblock-short"><p>Records a marker of the given duration that just ended.</p>
</div><div class="item-left module-item"><a class="fn" href="fn.set_profiler_hooks.html" title="webrender::profiler::set_profiler_hooks fn">set_profiler_hooks</a></div><div class="item-right docblock-short"><p>Set the profiler callbacks, or None to disable the profiler.
This function must only ever be called before any WR instances
have been created, or the hooks will not be set.</p>
</div><div class="item-left module-item"><a class="fn" href="fn.thread_is_being_profiled.html" title="webrender::profiler::thread_is_being_profiled fn">thread_is_being_profiled</a></div><div class="item-right docblock-short"><p>Returns true if the current thread is being profiled.</p>
</div></div></section><section id="search" class="content hidden"></section><div id="rustdoc-vars" data-root-path="../../" data-current-crate="webrender" data-search-index-js="../../search-index.js" data-search-js="../../search.js"></div>
    <script src="../../main.js"></script>
</body></html>