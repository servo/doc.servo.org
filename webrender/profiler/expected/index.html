<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><meta name="generator" content="rustdoc"><meta name="description" content="API documentation for the Rust `expected` mod in crate `webrender`."><meta name="keywords" content="rust, rustlang, rust-lang, expected"><title>webrender::profiler::expected - Rust</title><link rel="preload" as="font" type="font/woff2" crossorigin href="../../../SourceSerif4-Regular.ttf.woff2"><link rel="preload" as="font" type="font/woff2" crossorigin href="../../../FiraSans-Regular.woff2"><link rel="preload" as="font" type="font/woff2" crossorigin href="../../../FiraSans-Medium.woff2"><link rel="preload" as="font" type="font/woff2" crossorigin href="../../../SourceCodePro-Regular.ttf.woff2"><link rel="preload" as="font" type="font/woff2" crossorigin href="../../../SourceSerif4-Bold.ttf.woff2"><link rel="preload" as="font" type="font/woff2" crossorigin href="../../../SourceCodePro-Semibold.ttf.woff2"><link rel="stylesheet" type="text/css" href="../../../normalize.css"><link rel="stylesheet" type="text/css" href="../../../rustdoc.css" id="mainThemeStyle"><link rel="stylesheet" type="text/css" href="../../../ayu.css" disabled><link rel="stylesheet" type="text/css" href="../../../dark.css" disabled><link rel="stylesheet" type="text/css" href="../../../light.css" id="themeStyle"><script id="default-settings" ></script><script src="../../../storage.js"></script><script src="../../../crates.js"></script><script defer src="../../../main.js"></script>
    <noscript><link rel="stylesheet" href="../../../noscript.css"></noscript><link rel="alternate icon" type="image/png" href="../../../favicon-16x16.png"><link rel="alternate icon" type="image/png" href="../../../favicon-32x32.png"><link rel="icon" type="image/svg+xml" href="../../../favicon.svg"></head><body class="rustdoc mod"><!--[if lte IE 11]><div class="warning">This old browser is unsupported and will most likely display funky things.</div><![endif]--><nav class="mobile-topbar"><button class="sidebar-menu-toggle">&#9776;</button><a class="sidebar-logo" href="../../../webrender/index.html"><div class="logo-container"><img class="rust-logo" src="../../../rust-logo.svg" alt="logo"></div>
        </a><h2 class="location"></h2>
    </nav>
    <nav class="sidebar"><a class="sidebar-logo" href="../../../webrender/index.html"><div class="logo-container"><img class="rust-logo" src="../../../rust-logo.svg" alt="logo"></div>
        </a><h2 class="location"><a href="#">Module expected</a></h2><div class="sidebar-elems"><div class="block items"><ul><li><a href="#constants">Constants</a></li></ul></div><div id="sidebar-vars" data-name="expected" data-ty="mod" data-relpath="./"></div><script defer src="./sidebar-items.js"></script></div></nav><main><div class="width-limiter"><div class="sub-container"><a class="sub-logo-container" href="../../../webrender/index.html"><img class="rust-logo" src="../../../rust-logo.svg" alt="logo"></a><nav class="sub"><div class="theme-picker hidden"><button id="theme-picker" aria-label="Pick another theme!" aria-haspopup="menu" title="themes"><img width="18" height="18" alt="Pick another theme!" src="../../../brush.svg"></button><div id="theme-choices" role="menu"></div></div><form class="search-form"><div class="search-container"><div>
                                <input class="search-input" name="search" autocomplete="off" spellcheck="false" placeholder="Click or press ‘S’ to search, ‘?’ for more options…" type="search"></div><button type="button" id="help-button" title="help">?</button><a id="settings-menu" href="../../../settings.html" title="settings"><img width="18" height="18" alt="Change settings" src="../../../wheel.svg"></a></div></form></nav></div><section id="main-content" class="content"><div class="main-heading">
    <h1 class="fqn"><span class="in-band">Module <a href="../../index.html">webrender</a>::<wbr><a href="../index.html">profiler</a>::<wbr><a class="mod" href="#">expected</a><button id="copy-path" onclick="copy_path(this)" title="Copy item path to clipboard"><img src="../../../clipboard.svg" width="19" height="18" alt="Copy item path"></button></span></h1><span class="out-of-band"><a class="srclink" href="../../../src/webrender/profiler.rs.html#18-56" title="goto source code">source</a> · <a id="toggle-all-docs" href="javascript:void(0)" title="collapse all docs">[<span class="inner">&#x2212;</span>]</a></span></div><h2 id="constants" class="small-section-header"><a href="#constants">Constants</a></h2>
<div class="item-table"><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.ALPHA_PASSES.html" title="webrender::profiler::expected::ALPHA_PASSES constant">ALPHA_PASSES</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.AVG_BACKEND_CPU_TIME.html" title="webrender::profiler::expected::AVG_BACKEND_CPU_TIME constant">AVG_BACKEND_CPU_TIME</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.AVG_GPU_TIME.html" title="webrender::profiler::expected::AVG_GPU_TIME constant">AVG_GPU_TIME</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.AVG_IPC_TIME.html" title="webrender::profiler::expected::AVG_IPC_TIME constant">AVG_IPC_TIME</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.AVG_RENDERER_CPU_TIME.html" title="webrender::profiler::expected::AVG_RENDERER_CPU_TIME constant">AVG_RENDERER_CPU_TIME</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.CHANGED_TARGETS.html" title="webrender::profiler::expected::CHANGED_TARGETS constant">CHANGED_TARGETS</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.COLOR_PASSES.html" title="webrender::profiler::expected::COLOR_PASSES constant">COLOR_PASSES</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.CREATED_TARGETS.html" title="webrender::profiler::expected::CREATED_TARGETS constant">CREATED_TARGETS</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.DISPLAY_LIST_BUILD_TIME.html" title="webrender::profiler::expected::DISPLAY_LIST_BUILD_TIME constant">DISPLAY_LIST_BUILD_TIME</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.DISPLAY_LIST_MB.html" title="webrender::profiler::expected::DISPLAY_LIST_MB constant">DISPLAY_LIST_MB</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.DISPLAY_LIST_SEND_TIME.html" title="webrender::profiler::expected::DISPLAY_LIST_SEND_TIME constant">DISPLAY_LIST_SEND_TIME</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.DISPLAY_LIST_TOTAL_TIME.html" title="webrender::profiler::expected::DISPLAY_LIST_TOTAL_TIME constant">DISPLAY_LIST_TOTAL_TIME</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.DRAW_CALLS.html" title="webrender::profiler::expected::DRAW_CALLS constant">DRAW_CALLS</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.FONT_TEMPLATES_MB.html" title="webrender::profiler::expected::FONT_TEMPLATES_MB constant">FONT_TEMPLATES_MB</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.GPU_CACHE_BLOCKS_SAVED.html" title="webrender::profiler::expected::GPU_CACHE_BLOCKS_SAVED constant">GPU_CACHE_BLOCKS_SAVED</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.GPU_CACHE_BLOCKS_TOTAL.html" title="webrender::profiler::expected::GPU_CACHE_BLOCKS_TOTAL constant">GPU_CACHE_BLOCKS_TOTAL</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.GPU_CACHE_BLOCKS_UPDATED.html" title="webrender::profiler::expected::GPU_CACHE_BLOCKS_UPDATED constant">GPU_CACHE_BLOCKS_UPDATED</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.GPU_CACHE_ROWS_TOTAL.html" title="webrender::profiler::expected::GPU_CACHE_ROWS_TOTAL constant">GPU_CACHE_ROWS_TOTAL</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.GPU_CACHE_ROWS_UPDATED.html" title="webrender::profiler::expected::GPU_CACHE_ROWS_UPDATED constant">GPU_CACHE_ROWS_UPDATED</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.IMAGE_TEMPLATES_MB.html" title="webrender::profiler::expected::IMAGE_TEMPLATES_MB constant">IMAGE_TEMPLATES_MB</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.MAX_BACKEND_CPU_TIME.html" title="webrender::profiler::expected::MAX_BACKEND_CPU_TIME constant">MAX_BACKEND_CPU_TIME</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.MAX_GPU_TIME.html" title="webrender::profiler::expected::MAX_GPU_TIME constant">MAX_GPU_TIME</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.MAX_IPC_TIME.html" title="webrender::profiler::expected::MAX_IPC_TIME constant">MAX_IPC_TIME</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.MAX_RENDERER_CPU_TIME.html" title="webrender::profiler::expected::MAX_RENDERER_CPU_TIME constant">MAX_RENDERER_CPU_TIME</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.MAX_SCENE_BUILD_TIME.html" title="webrender::profiler::expected::MAX_SCENE_BUILD_TIME constant">MAX_SCENE_BUILD_TIME</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.NUM_FONT_TEMPLATES.html" title="webrender::profiler::expected::NUM_FONT_TEMPLATES constant">NUM_FONT_TEMPLATES</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.NUM_IMAGE_TEMPLATES.html" title="webrender::profiler::expected::NUM_IMAGE_TEMPLATES constant">NUM_IMAGE_TEMPLATES</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.NUM_RASTERIZED_BLOBS.html" title="webrender::profiler::expected::NUM_RASTERIZED_BLOBS constant">NUM_RASTERIZED_BLOBS</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.RASTERIZED_BLOBS_MB.html" title="webrender::profiler::expected::RASTERIZED_BLOBS_MB constant">RASTERIZED_BLOBS_MB</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.RENDERED_PICTURE_CACHE_TILES.html" title="webrender::profiler::expected::RENDERED_PICTURE_CACHE_TILES constant">RENDERED_PICTURE_CACHE_TILES</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.TEXTURE_DATA_UPLOADED.html" title="webrender::profiler::expected::TEXTURE_DATA_UPLOADED constant">TEXTURE_DATA_UPLOADED</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.TOTAL_PICTURE_CACHE_TILES.html" title="webrender::profiler::expected::TOTAL_PICTURE_CACHE_TILES constant">TOTAL_PICTURE_CACHE_TILES</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.TOTAL_PRIMITIVES.html" title="webrender::profiler::expected::TOTAL_PRIMITIVES constant">TOTAL_PRIMITIVES</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.USED_TARGETS.html" title="webrender::profiler::expected::USED_TARGETS constant">USED_TARGETS</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.VERTICES.html" title="webrender::profiler::expected::VERTICES constant">VERTICES</a></div><div class="item-right docblock-short"></div></div><div class="item-row"><div class="item-left module-item"><a class="constant" href="constant.VISIBLE_PRIMITIVES.html" title="webrender::profiler::expected::VISIBLE_PRIMITIVES constant">VISIBLE_PRIMITIVES</a></div><div class="item-right docblock-short"></div></div></div></section><section id="search" class="content hidden"></section></div></main><div id="rustdoc-vars" data-root-path="../../../" data-current-crate="webrender" data-themes="ayu,dark,light" data-resource-suffix="" data-rustdoc-version="1.60.0-nightly (5e57faa78 2022-01-19)" ></div>
</body></html>